src/core/hle/service/am/am.cpp

```diff
@@ -97,6 +101,16 @@ void IWindowController::GetAppletResourceUserId(HLERequestContext& ctx) {
     rb.Push<u64>(process_id);
 }
 
+void IWindowController::GetAppletResourceUserIdOfCallerApplet(HLERequestContext& ctx) {
+    const u64 process_id = 0;
+
+    LOG_DEBUG(Service_AM, "(STUBBED) called");
```

Comment:
unless they are spammed, stubbed logs should be warnings

```suggestion
    LOG_WARNING(Service_AM, "(STUBBED) called");
```

Reply:
Narr already told me this.
I will check again just to be sure but most of the lines are big spammers when the controller applet is open.